plugins/kubernetes/src/components/KubernetesDrawer/KubernetesDrawer.tsx

```diff
@@ -136,6 +144,19 @@ const KubernetesDrawerContent = <T extends KubernetesDrawerable>({
         </IconButton>
       </div>
       <div className={classes.options}>
+        <div>
+          {clusterLink && (
+            <Button
+              variant="contained"
+              color="primary"
+              size="small"
+              href={clusterLink}
+              target="_blank"
+            >
```

Comment:
could you try doing

```tsx
import { Link } from '@backstage/core-components';

<Button variant="contained" color="primary" size="small" component={Link} to={clusterLink}>
```

This way it looks like a MUI button but behaves like a native Backstage `Link`. The benefit of that, is that we make sure that it opens a new page only if it's an external link (or can make that configurable per org or even per user), we apply safety rules like `noopener`, will apply metrics / click tracking uniformly when applicable ...